### ISSUE TITLE
Only one file included in "errors" variable inside lu:errors

### ISSUE THREAD

**Issue by dhardtke** [burt202/lite-uploader#22]
Before I could iterate over the errors variable like this:
```javascript
var html = "<ul>";
$.each(errors, function (key, value) {
	html += "<li>" + value.name + ": ";

	$.each(value.errors, function (i, err) {
		html += lang["FILEUPLOAD_MESSAGES"][err.type.toUpperCase()];

		// append <br /> if necessary
		html += i == value.errors.length - 1 ? "" : "<br />";
	});
});
html += "</ul>";
```

Now with the most recent version I can't anymore.

If I console.log my errors variable I get to see this:
`({name:"Foto0928.jpg", errors:[{type:"size", rule:200000, given:338968}]})`

I don't get that. Even if I select multiple files, I only get one file included in that "object".
What's even weirder is that I'm wondering why this object has curved brackets around it. I thought it'd be just a plain object?

Either way I'm doing something wrong or there's a bug in this library.

Thanks.

**Comment by burt202** [comment]
Can you send over a full, simplified example of the problem youre seeing and ill take a look? Cheers

**Comment by dhardtke** [comment]
Sure: https://jsfiddle.net/csv4mqjm/

If you select multiple files in the dialog that pops up and there is more than one validation error, you only get to see one in the console.

And I don't know how to properly iterate over the resulting errors object (because it is clearly not a list or smth. like that).

**Comment by burt202** [comment]
OK I remember this now. From: http://api.jquery.com/on/

> If an array was passed to the second argument of .trigger() or .triggerHandler(), each element in the array will be presented to the event handler as an individual parameter.

So basically, all the errors are in the errors callback just as individual params, so using arguments and striping off the first param would be your immediate fix.

I could also make a fix in the source where I wrap the errors array in another array (I think I had this before) so they work as you desire but I'd have to give it some thought.

Do you have any preference?

If you're happy with using arguments then I'll update the documentation to make this clearer

Cheers

**Comment by dhardtke** [comment]
I would prefer the second solution because it seems more natural to me to have a list (aka an array) of errors.

**Comment by burt202** [comment]
Ok, well I'm guessing youre unblocked for now? Let me give it some thought, and Ill get back to you

**Comment by dhardtke** [comment]
It's fine, I'm using lite-uploader on my development site, so it's not used in production yet.

I look forward to hearing from you

**Comment by burt202** [comment]
Fixed in v3.3.0. Close this issue if youre happy with it.

Cheers

**Comment by dhardtke** [comment]
Perfect. Thank you!
